src/javascript/serverSettings/backgroundJobs/index.jsx

```diff
@@ -1,39 +1,61 @@
-import React, {useState} from 'react';
-import {Header, LayoutContent, Paper, Tab, TabItem} from '@jahia/moonstone';
+import React, {useMemo, useRef, useState} from 'react';
+import {Button, Header, LayoutContent, Paper, Tab, TabItem} from '@jahia/moonstone';
 import HistoryBackgroundJobsTable from './HistoryBackgroundJobsTable';
 import ScheduledBackgroundJobsTable from './ScheduledBackgroundJobsTable';
 import {useTranslation} from 'react-i18next';
+import SvgReload from '@jahia/moonstone/dist/icons/components/Reload';
```

Comment:
hard-coded dependency paths should be avoided if possible. I see that we have a Reload icon available for import https://jahia.github.io/moonstone/?path=/docs/tokens-icons--docs. 

Possible to use that here instead? We also have an SvgWrapper  moonstone component if needed.